<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>35</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	font: 63 35pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
    </property>
    <property name="text">
     <string>Your Cart</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="cartProducts">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>159</number>
    </attribute>
    <column>
     <property name="text">
      <string>Item Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>1 x Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Amount</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="totalAmount">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>12</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
    </property>
    <property name="text">
     <string>Total Amount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkOutButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 15px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Check Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>54</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 15px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Add Quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 15px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Subtract Quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 15px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 15px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
